docs/sources/reference/api/docker_remote_api.md

```diff
@@ -71,15 +71,33 @@ This endpoint now returns `SystemTime`, `HttpProxy`,`HttpsProxy` and `NoProxy`.
 
 ### What's new
 
+**New!**
```

Comment:
The build supports `LABEL` command. Use this add metadata
to an image. For example you could add data describing the content of an image.

`LABEL "com.example.vendor"="ACME Incorporated"`